### ISSUE TITLE
Help navigation NullPointerException

### ISSUE THREAD

**Issue by ChrisMGeiger** [NationalSecurityAgency/ghidra#493]
**Describe the bug**
NullPointerException when accessing or navigating Help

**To Reproduce**
1. Open Help
2. Click on a topic (in tree view or find view)
3. NullPointerException (trace below)

**Expected behavior**
Navigation without uncaught exceptions

**Environment (please complete the following information):**
 - OS: Windows 10 Pro
 - Java Version: 11.0.2
 - Ghidra Version: 9.0.2

**Additional context**
Uncaught exception occurs on first opening help and subsequent navigation between topics after they are rendered in the right hand pane.

[ghidra_help_exception_20190423.txt](https://github.com/NationalSecurityAgency/ghidra/files/3107527/ghidra_help_exception_20190423.txt)

**Comment by ryanmkurtz** [comment]
Were you using our official release or did you build it yourself?

**Comment by dragonmacher** [comment]
It appears from the exception that the Help system did not initialize correctly.  Does the Ghidra log file indicate any issues when launching Ghidra?   The log file is located at `<user home>/.ghidra/.ghidra-<version>/application.log`

**Comment by ChrisMGeiger** [comment]
@ryanmkurtz: 9.0.2 official build
@dragonmacher: database came as an import of a very large database from IDA.
```
100k+  (CParserUtils) Invalid signature: unable to isolate function name
3000+ (FunctionsXmlMgr) Failed to create function
158 (FunctionDB) WARNING! Adding overlapping local variable
6: NullPointerException in ghidra.program.database.references.ReferenceDBManager.referenceAdded - with traceback - see below
~120: NullPointerException without tracebacks
```
After that, first messages of consequence come after first check in to the server. The recurring NullPointerException on help, didn't occur until I used help days later.

[ghidra_help_exception_application_log_startup_NullPointerException_20190423.txt](https://github.com/NationalSecurityAgency/ghidra/files/3108735/ghidra_help_exception_application_log_startup_NullPointerException_20190423.txt)

[ghidra_help_exception_application_log_post_checkin_NullPointerException_20190423.txt](https://github.com/NationalSecurityAgency/ghidra/files/3108736/ghidra_help_exception_application_log_post_checkin_NullPointerException_20190423.txt)

**Comment by dragonmacher** [comment]
I was unable to reproduce the problem, but I have put in a fix that could cause the issue under the right conditions.

It should be available in 9.1.
